lock-api/src/main/java/com/palantir/lock/LockDescriptor.java

```diff
@@ -48,25 +48,30 @@
     }
 
     @JsonIgnore
+    @SuppressWarnings("checkstyle:RegexpSinglelineJava") // JDK StandardCharsets is only a feature in 1.7+
     public String getLockIdAsString() {
         return new String(bytes, Charsets.UTF_8);
     }
 
     @Override
-    public int compareTo(LockDescriptor o) {
-        return UnsignedBytes.lexicographicalComparator().compare(this.bytes, o.bytes);
+    public int compareTo(LockDescriptor other) {
+        return UnsignedBytes.lexicographicalComparator().compare(this.bytes, other.bytes);
     }
 
     @Override
     public String toString() {
         String lockIdAsString = getLockIdAsString();
-        return getClass().getSimpleName() + " [" +
-                (BASIC_PRINTABLE_ASCII.matchesAllOf(lockIdAsString) ?
-                        lockIdAsString :
-                        BaseEncoding.base16().encode(bytes))
+        return getClass().getSimpleName() + " ["
+                + getLockIdString(lockIdAsString)
                 + "]";
     }
 
+    private String getLockIdString(String lockIdAsString) {
```

Comment:
Yes, agreed that `encodeToAscii` is a better name. For the 2 methods: the `getLockIdAsString()` is public API that returns a string which may or may not be printable (in that it may contain ASCII control characters); this one is an internal method called by `toString()` that's guaranteed to return something printable.